simple/msg.c

```diff
@@ -260,6 +260,7 @@ static int server_connect(void)
 	}
 
 	/* Open the endpoint */
+	info->tx_attr->op_flags = FI_REMOTE_COMPLETE;
```

Comment:
For some simple tests, ft_finalize is not called but FI_REMOTE_COMPLETE is set. Do we want the simple tests to follow this coding structure or all of them should include ft_finalize with corresponding recv-post? @shefty